tests/tensorflow2/test_tensorflow2_datatypes.py

```diff
@@ -28,5 +29,7 @@ def test_tensorflow2_datatypes():
         try:
             _get_proto_dtype(np.dtype(_type))
         except Exception:
-            assert False, f"{_type} not supported"
+            if _NP_TO_TF[_type] != tf.bfloat16:
+                # bfloat16 is only supported on TPUs.
```

Comment:
Need more context. Do none of the SageMaker instance types have TPUs? So, is `bfloat16` not supported on any training jobs in SageMaker?